src/routers/user.js

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import {
+  onAddUser,
+  onUpdateUser,
+  onGetUser,
+  onDeleteUser,
+  onGetAutosuggestedUsers,
+} from '../controllers/user.js';
+import { validateUserQuery, validateSchema } from '../middleware/validation.js';
+import { userPostSchema, userPutSchema } from '../schemas/user.js';
+
+const router = Router();
+
+router
+  .route('/')
+  .get(validateUserQuery, onGetAutosuggestedUsers)
+  .post(validateSchema(userPostSchema), onAddUser)
+  .put(validateSchema(userPutSchema), onUpdateUser);
```

Comment:
It is better to propagate id in url, like `PUT /users/:id` thus while reading the API documentation it will be clear that this endpoint updates user with specific ID. Otherwise it is not so obvious what actually this endpoint  does. 
Also maybe we will need endpoint for bulk update of all users in that case `PUT /users` endpoint will be a good choice

Reply:
Done